src/routes/+page.svelte

```diff
@@ -1,18 +1,47 @@
 <script lang="ts">
 	import Card from '$lib/components/Card.svelte';
 	import type { PageData } from './$types';
-	import SearchBar from './SearchBar.svelte';
 
 	export let data: PageData;
+	let searchQuery = '';
+
+	const onSearch = (event: Event) => {
+		event.preventDefault();
+		const query = new URLSearchParams({ query: searchQuery }).toString();
+		window.location.href = `?${query}`;
+	};
+
+	const resetSearch = (event: Event) => {
```

Comment:
I think the standard in Svelte is to use the function syntax instead of the arrow syntax.
```ts
function resetSearch(event: Event) {
  ...
}
```